docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+                    ...
+                },
+                "options": {
+                    "scoringFormula" : string,
+                    "readonly" : boolean,
+                    "hideInSurvey" : boolean,
+                },
+                "rules": {
+                    ...
+                }
+        ...
+}
+```
+
+# 4.0: Helper Elements
+
+A helper object represents a non-input, layout-related front-end element. Each type of helper may
+contain type specific options. The different types and their options are described in the section
+[4.0: Helper Types](#4.0:-helper-types) below. 
+
+In general, a helper object has the same format as a field:
+
+```js
+{
+    "schema" : {
+        ...
+        "helpers" : {
+            "nameOfHelper" : {
+                "type": string,
+                "description": {
+                    "lang1": string,
+                    "lang2": string
+                },
+                "options": {
+                    ...
+                },
+                "rules": {
+                    "requireIf": string, /* LORIS Logic Parser formula */
+                    "displayIf": string, /* LORIS Logic Parser formula */
+                    "disableIf": string, /* LORIS Logic Parser formula */
+                }
+        ...
+}
+```
+
+`type`: Required string.
+        It represents the helper's data type i.e. static, section, table, etc.
+
+`description`: Required string.
+               The human readable description of this helper, provided in every available language,
+               and also the text to be displayed in the instrument.
+
+It may be rendered on the front-end as a header, depending on the helper type i.e. this will be the
+content of the static helper itself, the header of the section helper, or column header for the table.
+
+`options`: Required object of type-specific keys.
+           It contains the type-dependent options for this helper, further described in section
+           [4.0: Helper Types](#4.0:-helper-types).
+
+`rules`: Required object with keys whose value may be an empty string as described for `fields`.
+While instrument `fields` have corresponding `data` associated with it upon form submission, a `helper` is rendered on the front-end as an integral element that however does not collect data. There are
+5 types of helpers: the static element, and 4 other layout-specific formatting elements, group, row, table, and section.
+
+## 4.1: Helper Types
+
+### 4.1.1: Static
+
+A `helper` of type "static" represents headings, sub-headings, labels, or informational text that is required in
+an instrument. These can be rendered on the front-end as a header, static, or link element.
+Static helpers are some text displayed to provide information and without an accompanying data input.
+Static helpers do not have an `options` key.
+
+```js
+{
+    "schema" : {
+        ...
+        "helpers" : {
+            "nameOfHelper" : {
+                "type": "static",
```

Comment:
What does a "static" type mean?

Reply:
label, heading subheading ... anything text that does not get submitted

Reply:
I would call it something like "statictext", then.